apps/web/global.css

```diff
@@ -2,28 +2,6 @@
 @import 'tailwindcss/components';
 @import 'tailwindcss/utilities';
 
-@layer base {
```

Comment:
what's our new font choice?

Reply:
Matches the venice.is site:
- Montserrat for primary
- some monospace (e.g. Source Code Pro is fine) for code or technical areas